lib/customLogger.js

```diff
@@ -30,20 +30,24 @@ function generateCustomLog(format, message, config) {
 // Function to write log entry to stdout
 function logToStdOut(log, format, spacing, config) {
     // Set standard output stream
-    var outputStream = process.stdout; 
+    var outputStream = process.stdout;
 
     if (format === 'json') {
-        var stringifiedLog = utils.stringifyLog(log, format, spacing),
+        var stringifiedLog = utils.stringifyLog(log, format, spacing);
+        // no need for colorizedLog if compact options is enabled
```

Comment:
Hi @sirius226, can you please explain the idea behind disabling the need for a colorized log (in the output stream) in case of compact output? I think it's better to have a clear identifier in the console for better visibility.

Reply:
Hi @ArunMichaelDsouza, 
I think for most use cases, compact format is more for shipping logs to log aggregation system like ELK, colorized is not needed here.

Also  colorizeStdoutFormat will use this regex: `CUSTOMLEVELJSON: /"level": "(.*?)"/` to fetch the level, which will fail for compact format, since there is no space between key and value

Reply:
Yes, that is true, I might have to change it.

Reply:
Thanks again for the PR, the changes will be released in the next version!